api/configs/middleware/vdb/analyticdb_config.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class AnalyticdbConfig(BaseModel):
+    """
+    Analyticdb configs
```

Comment:
Any possible official ADB docs to refer for the definitions or format of the concept and values?